Library/Formula/sngrep.rb

```diff
@@ -0,0 +1,23 @@
+class Sngrep < Formula
+  homepage "https://github.com/irontec/sngrep"
+  url "https://github.com/irontec/sngrep/archive/v0.2.2.tar.gz"
```

Comment:
yup, I'm not in any rush to get it in. I'll go ahead and update the PR when release comes.